fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -65,9 +66,16 @@ fun TabBar(
         ) {
             tabDataList.forEachIndexed { index, tabData ->
                 tabData.selected = index == selectedIndex
+                var accessibilityDescriptionValue = if(tabData.accessibilityDescription != null) { tabData.accessibilityDescription }
+                                               else{ tabData.title + if(tabData.selected) ": Active" else ": Inactive" }
```

Comment:
can we add this strings active and inactive to the strings file, so that they are localized correctly